src/any_agent/tracing/instrumentation/llama_index.py

```diff
@@ -0,0 +1,73 @@
+# mypy: disable-error-code="no-untyped-call, no-untyped-def"
```

Comment:
Why are these disabled?

Reply:
Not sure if they should be disabled in this file 🤔 I was using those in the modules that use `wrapt`